control/robust.py

```diff
@@ -136,26 +141,26 @@ def hinfsyn(P,nmeas,ncon):
 
     """
 
-    #Check for ss system object, need a utility for this?
+    # Check for ss system object, need a utility for this?
 
-    #TODO: Check for continous or discrete, only continuous supported right now
-        # if isCont():
-        #    dico = 'C'
-        # elif isDisc():
-        #    dico = 'D'
-        # else:
+    # TODO: Check for continous or discrete, only continuous supported right now
```

Comment:
We should raise an exception if a discrete time system is passed as an argument.  Use `isdtime(P, strict=True)` to check.